src/frontend/nginx.conf

```diff
@@ -5,11 +5,16 @@ server {
     location / {
         root   /usr/share/nginx/html;
         index  index.html index.htm;
-        try_files $uri /index.html;                 
-    }
+        try_files $uri /index.html;
 
-    error_page   500 502 503 504  /50x.html;
-    location = /50x.html {
-        root   /usr/share/nginx/html;
+        expires -1;
+        add_header 'Cache-Control' 'no-store, no-cache, must-revalidate, proxy-revalidate, max-age=0';
+        add_header X-Content-Type-Options "nosniff" always;
+        add_header X-XSS-Protection "1; mode=block";
+        add_header Pragma "no-cache";
+        add_header Content-Security-Policy "default-src 'self';";
```

Comment:
we use bootstrap from cdn.  This might block it.

Reply:
Great catch! Thank you.

Reply:
Added an exception to the exact cdn paths for both stylesheet and js files. Not adding wildcard explicitly to eradicate loopholes. The only downside is we need to update the path, when we upgrade bootstrap in the future. Let me know if that addresses your concern.

I am merging this now to allow tomorrow morning's zap scan to pick up these changes. Feel free to further comment here or chat, if there's any additional convo needed. Else, please mark this thread resolved.